content/deployment/quickstart.md

```diff
@@ -31,15 +74,13 @@ common to RHEL based operating systems, you may have to adjust the
 Apache path used to match your installation.
 
 ```bash
-mkdir /etc/httpd/modsecurity.d
-mv coreruleset-master.zip /etc/httpd/modsecurity.d/
-cd /etc/httpd/modsecurity.d
-unzip coreruleset-master.zip -d owasp-modsecurity-crs
+$ mkdir /etc/httpd/modsecurity.d
+$ tar zxvf v{{< param crs_latest_release >}}.tar.gz -C /etc/httpd/modsecurity.d/owasp-modsecurity-crs
```

Comment:
```suggestion
$ tar -zxvf v{{< param crs_latest_release >}}.tar.gz -C /etc/httpd/modsecurity.d/owasp-modsecurity-crs
```

Reply:
Technically, the dash is not needed....

Reply:
In that case I don't really mind (I prefer the *nix style flags but).

Reply:
`tar` must be one of those tools that just depends on how (and when?) you learn it :)

I would just do `tar xvf ...` personally, but out of `tar zxvf` and `tar -zxvf` I'd choose to document it without the dash. I think that's the most common invocation, nowadays?

Reply:
Really? As I said, according to the man pages, the *nix style flags (with dash) are considered more modern, some implementations consider the compound style (without dash) a legacy format.

Reply:
Done. dash for the win.